Source/RxSwift/mapMany.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  mapMany.swift
+//  RxSwiftExt
+//
+//  Created by Joan Disho on 06.05.18.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import RxSwift
+
+extension ObservableType where E: Collection {
+
+    /**
+     Projects elements of each element of an observable collection into a new form.
+     - parameter transform: A transform function to apply to each source element.
+     - returns: An observable collection whose elements are the result of invoking the transform function on each element of source.
+     */
+
+    public func mapMany<T>(_ transform: @escaping (E.Element) -> T) -> Observable<[T]> {
```

Comment:
I think the `transform` block needs to throw like `map` itself: https://github.com/ReactiveX/RxSwift/blob/master/RxSwift/Observables/Map.swift#L20

What about the implementation example I put in https://github.com/RxSwiftCommunity/RxSwiftExt/issues/93#issuecomment-378631505 ?

Reply:
It is a bit unclear why it should throw 🤔

Reply:
Because the original map method accepts a throwing closure, so we should accept a throwing closure as well:
func map<T>(_ transform: (Self.Element) **throws** -> T) rethrows -> [T]